app/javascript/controllers/autocomplete_edit_controller.js

```diff
@@ -9,7 +9,6 @@ export default class extends Controller {
 
   change() {
     if(this.valueTarget.value) {
-      this.inputTarget.readOnly = true
```

Comment:
Fields that are read only will not validate (see [willValidate](https://developer.mozilla.org/en-US/docs/Web/API/HTMLObjectElement/willValidate)).